tests/python/gaia-ui-tests/gaiatest/apps/gallery/app.py

```diff
@@ -21,6 +21,7 @@ class Gallery(Base):
     name = 'Gallery'
 
     _gallery_items_locator = (By.CSS_SELECTOR, 'div.thumbnail')
+    _overlay_locator = (By.ID, 'overlay')
```

Comment:
Yes. now that we actually hide the main sections when the overlay is displayed, you can't depend on the thumbnail view when there are no photos. You need to wait for the overlay. So I updated the tests since my changes "break" them.